### DISCUSSION TITLE
ERROR: Unknown command: pair_styled deepmd

### DISCUSSION THREAD

**Discussion by Coldwater-fish** [deepmodeling/deepmd-kit#2212] [Q&A] [answered]
I installed deepmd-kit via offline package deepmd-kit-2.1.5-cpu-Linux-x86_64.sh. When I run  deepmd-kit/bin/lmp using a trained model,  it leads to the following error:

> ERROR: Unknown command: pair_styled     deepmd    graph-compress.pb (src/input.cpp:274)
> Last command: pair_styled     deepmd     graph-compress.pb

I also tried the deepmd-kit-2.1.1-cpu version, and the same error occurred. 
And when execute  **lmp -h** , there is no **deepmd** under **Pair styles**. 
(I have done it : source /path/to/deepmd-kit/bin/activate /path/to/deepmd-kit)

How to solve this problem? Thanks a lot!

**Comment by Coldwater-fish** [comment]
Also tried **conda activate /path/to/deepmd-kit**. It still does't work.

**Comment by njzjz** [comment, marked as answer]
The correct command is `pair_style` instead of `pair_styled`. You can refer to LAMMPS documentation - `pair_style` command is not provided by us.

**Comment by jic198** [comment]
Hello. I am having a similar problem. `ERROR: Unrecognized pair style 'deepmd' (src/force.cpp:279)`. I used `conda install lammps cudatoolkit=11.6 horovod -c https://conda.deepmodeling.com` to install the lammps. I used `conda activate deepmd` to activate my deepmd env before I ran `lmp -i in.lammps`. So I think I've followed all the necessary steps. Do you know what could be the problem? Thank you so much!

**Reply by njzjz** [reply]
Did you install `libdeepmd`?

**Reply by jic198** [reply]
Yes I did. I can find it in `conda list`. `libdeepmd                 2.0.3            0_cuda11.3_gpu    https://conda.deepmodeling.com` I also have `libdeepmd_lmp.so` and `libdeepmd_lmp_low.so` in `deepmd/lib`. `deepmd` is the name of my conda env.

**Reply by njzjz** [reply]
2.0.3 does not match the latest lammps. You need to upgrade it to the latest version.

**Reply by jic198** [reply]
Thanks for the reply. What's the latest version of `libdeepmd`? I tried `conda update libdeepmd` still got 2.0.3

**Reply by jic198** [reply]
BTW, I am running GPU version of deepmd

**Reply by njzjz** [reply]
The latest version is 2.1.5
